JSDemos/Demos/Charts/PieWithAnnotations/description.md

```diff
@@ -0,0 +1,9 @@
+Annotations are containers for images, text blocks, and custom content that provide additional information about the visualized data.
```

Comment:
Similar comment as above or in other merge request ("provide"). A lot of this text repeats what I already commented on - can you please review those comments and apply changes. Thanks.